src/nwnsdk/nwn_client.py

```diff
@@ -74,3 +82,6 @@ def db_client(self) -> PostgresClient:
     @property
     def broker_client(self) -> RabbitmqClient:
         return self.rabbitmq_client
```

Comment:
Could these two properties be deleted? It would be better if only nwn_client functions are called.

Reply:
That is okay, but then I will refactor NWNClient to use RabbitMQClient and PostgresClient as mixins. That way, we do not have to write so many useless functions that only call the same function on the proper client. Will save us a lot of hassle ^^